tracer/src/Datadog.Trace/Processors/ObfuscatorTraceProcessor.cs

```diff
@@ -0,0 +1,58 @@
+// <copyright file="ObfuscatorTraceProcessor.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Processors;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.TraceProcessors
+{
+    internal class ObfuscatorTraceProcessor : ITraceProcessor
+    {
+        private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<ObfuscatorTraceProcessor>();
+
+        public ObfuscatorTraceProcessor()
+        {
+            Log.Information("ObfuscatorTraceProcessor initialized.");
```

Comment:
(nit - you can ignore as this is what is done for other processors.)
But I would made this a debug log